src/test/java/api/loans/ResolveClaimAsMissingApiTest.java

```diff
@@ -0,0 +1,114 @@
+      .getId().toString();
+  }
+
+  @Test
+  public void canResolveClaimedReturnedItemAsMissing() {
+    final String comment = "testing";
+
+    loansFixture.claimItemReturned(new ClaimItemReturnedRequestBuilder()
+      .forLoan(loanId)
+      .withItemClaimedReturnedDate(DateTime.now()));
+
+    final Response response = loansFixture
+      .resolveClaimAsMissing(new ResolveClaimAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment(comment));
+
+    assertLoanAndItem(response, comment);
+  }
+
+  @Test
+  public void cannotResolveClaimAsMissingWhenItemHasNotClaimedReturnStatus() {
+    final Response response = loansFixture
+      .attemptResolveClaimAsMissing(new ResolveClaimAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment("testing"));
+
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson(), hasErrorWith(hasMessage("Item is not Claimed returned")));
+    assertThat(response.getJson(), hasErrorWith(hasParameter("id", item.getId().toString())));
+  }
+
+  @Test
+  public void cannotResolveClaimAsMissingWhenLoanIsClosed() {
+    loansFixture.checkInByBarcode(item);
+
+    final Response response = loansFixture
+      .attemptResolveClaimAsMissing(new ResolveClaimAsMissingRequestBuilder()
+        .forLoan(loanId)
+        .withComment("testing"));
+
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson(), hasErrorWith(hasMessage("Loan is closed")));
+    assertThat(response.getJson(), hasErrorWith(hasParameter("id", loanId)));
+  }
+
+  @Test
+  public void cannotResolveClaimAsMissingWhenCommentIsNotProvided() {
+    final Response response = loansFixture
+      .attemptResolveClaimAsMissing(new ResolveClaimAsMissingRequestBuilder()
+        .forLoan(loanId));
+
+    assertThat(response.getStatusCode(), is(422));
+    assertThat(response.getJson(),
+      hasErrorWith(hasMessage("Comment is a required field")));
+    assertThat(response.getJson(), hasErrorWith(hasParameter(COMMENT, null)));
+  }
+
+  @Test
+  public void cannotResolveClaimAsMissingWhenLoanIsNotFound() {
+    final String notExistentLoanId = UUID.randomUUID().toString();
+
+    final Response response = loansFixture
+      .attemptResolveClaimAsMissing(new ResolveClaimAsMissingRequestBuilder()
+        .forLoan(notExistentLoanId)
+        .withComment("testing"));
+
+    assertThat(response.getStatusCode(), is(404));
+  }
+
+  private void assertLoanAndItem(Response response, String comment) {
+    JsonObject actualLoan = loansClient.getById(UUID.fromString(loanId)).getJson();
+    JsonObject actualItem = actualLoan.getJsonObject("item");
+
+    assertThat(response.getStatusCode(), is(204));
+    assertThat(actualItem, hasStatus("Missing"));
+    assertThat(actualLoan, isClosed());
+    assertThat(actualLoan, hasLoanProperty(ACTION, "markedMissing"));
+    assertThat(actualLoan, hasLoanProperty(ACTION_COMMENT, comment));
+  }
```

Comment:
This appears to be only used once, meaning that I believe the extraction is to help describe the intent. I think there are two separate intents lurking in this method, that the loan is closed and that the item is now considered missing.

Maybe this could instead be split into two methods `assertLoanClosed` and `assertItemMarkedMissing` 

(Personally, I prefer check over assert in these method names, however it seems that might lead to static analysis warnings)